guides/3)building_with_blocks/3)state_in_blocks.md

```diff
@@ -19,11 +19,11 @@ $demo_hangman
 
 Let's see how we do each of the 3 steps listed above in this game:
 
-1. We store the used letters in `used_letters_var`. In the constructor of `Variable`, we set the initial value of this to `[]`, an empty list. 
+1. We store the used letters in `used_letters_var`. In the constructor of `State`, we set the initial value of this to `[]`, an empty list. 
 2. In `btn.click()`, we have a reference to `used_letters_var` in both the inputs and outputs.
-3. In `guess_letter`, we pass the value of this `Variable` to `used_letters`, and then return an updated value of this `Variable` in the return statement.
+3. In `guess_letter`, we pass the value of this `State` to `used_letters`, and then return an updated value of this `State` in the return statement.
 
-With more complex apps, you will likely have many Variables storing session state in a single Blocks app.
+With more complex apps, you will likely have many State variables storing session state in a single Blocks app.
```

Comment:
In the `interface_state.md` guide should we mention that interfaces only support a single state variable? And point them to blocks for more complex state?

Reply:
Sure will make the addition